cmd/agent/dist/conf.d/postgres.d/conf_aws_aurora.yaml.default

```diff
@@ -0,0 +1,46 @@
+ad_identifiers:
```

Comment:
Don't you miss the one for MySQL?

Reply:
I'll add a file for mysql as well

Reply:
Files should either here or in `integrations-core` but not in both.

Reply:
Ah okay, gotcha